docs/source/models/AOFDTD.rst

```diff
@@ -434,6 +434,30 @@ Usage
 The field solver can be chosen and configured in :ref:`fieldSolver.param <usage-params-core>`.
 
 
+Substepping
+-----------
+Any field solver can be combined with substepping in time.
+In this case, each iteration of the main PIC loop involves multiple invocations of the chosen field solver.
+Substepping is fully compatible with other numerics, such as absorbers, incident field, laser generation.
+A substepping field solver has the same orders of accuracy in spatial and time steps as the base solver.
+
+A user sets main PIC time step value as usual in :ref:`grid.param <usage-params-core>`, and selects the number of field solver substeps via template argument `T_numSubsteps`.
+Field solver will internally operate with :math:`\Delta t_{sub} = \Delta t / T_numSubsteps`.
```

Comment:
'T_numSubsteps' -> 'T_{numSubsteps}'

Reply:
It's the name of the template parameter of that solver.